api/proto/relay/relay.proto

```diff
@@ -34,7 +34,7 @@ message GetChunksRequest {
 
   // If this is an authenticated request, this should hold the ID of the requester. If this
   // is an unauthenticated request, this field should be empty.
-  uint64 requester_id = 2;
+  bytes requester_id = 2;
 
   // If this is an authenticated request, this field will hold a signature by the requester
   // on the chunks being requested.
```

Comment:
Document how this signature should be computed

Reply:
```
  // If this is an authenticated request, this field will hold a BLS signature by the requester
  // on the hash of this request. Relays may choose to reject unauthenticated requests.
  //
  // The following describes the schema for computing the hash of this request
  // This algorithm is implemented in golang using relay.auth.HashGetChunksRequest().
  //
  // All integers are encoded as unsigned 4 byte big endian values.
  //
  // Perform a keccak256 hash on the following data in the following order:
  // 1. the operator id
  // 2. for each chunk request:
  //    a. if the chunk request is a request by index:
  //       i. the blob key
  //       ii. the start index
  //       iii. the end index
  //    b. if the chunk request is a request by range:
  //       i. the blob key
  //       ii. each requested chunk index, in order
  bytes operator_signature = 3;
```